app/src/main/java/io/github/gmathi/novellibrary/util/Constants.kt

```diff
@@ -97,6 +94,24 @@ object Constants {
         const val RVN_HISTORY = "recentlyViewNovelsHistory"
     }
 
+    const val SIMPLE_NOVEL_BACKUP_FILE_NAME = "SimpleNovelBackup.txt"
+    const val DATABASES_DIR = "databases"
+    const val FILES_DIR = "files"
+    const val SHARED_PREFS_DIR = "shared_prefs"
+    const val DATA_SUBFOLDER = """/data/${BuildConfig.APPLICATION_ID}"""
+
+    const val WORK_KEY_RESULT = "result"
+
+    private class NotificationId {
```

Comment:
To prevent conflict in edge cases. Just as an example, say an automatic backup kicks in, but you also do a manual backup at the same time to another location. This way, you will see to different progress notifications. Otherwise, they would constantly overwrite each other.
It's just to be on the safe side.

PS. also, if you are an idiot like me that once forgot I was already using some id, and used it again lol  :P